giskard-server/src/main/java/ai/giskard/domain/Role.java

```diff
@@ -22,11 +21,13 @@ public class Role implements Serializable {
     @lombok.Getter
     @NotNull
     @Id
+    @GeneratedValue
     private int id;
 
     @lombok.Setter
     @lombok.Getter
     @NotNull
+    @Unique
     @Size(max = 50)
     @Column(length = 50)
```

Comment:
Are you sure you wanted a @Unique annotation from checkerframework and not just @Column(length = 50, unique=true)?

Reply:
yes changing this